src/app/screens/SignUp/index.test.tsx

```diff
@@ -10,44 +15,106 @@ const MINIMUM_ALERT_LENGTH = 1;
 const FORM_BUTTONS_AMOUNT = 2;
 const PASSWORD = 'test';
 const WRONG_CONFIRM_PASSWORD = 'wrongPassword';
+const API_URL = `${process.env.REACT_APP_API_BASE_URL}/users`;
+const STATUS_CREATED = 201;
+
+const server = setupServer(
+  rest.post(`${API_URL}`, (req, res, ctx) =>
+    res(ctx.status(STATUS_CREATED), ctx.json(OK_SIGNUP_API_RESPONSE))
+  )
+);
+
+afterEach(() => server.resetHandlers());
+afterAll(() => server.close());
 
 describe('Signup test', () => {
   test('Should match snapshot', () => {
-    const { container } = render(<SignUp />);
+    const { container } = render(
+      <Router>
+        <SignUp />
+      </Router>
+    );
```

Comment:
for this and every test, you could do this on `beforeEach` and have a `describe` scoped variable container that initially is `let container = null;` then you fill it on `beforeEach`
```
beforeEach(()=> {
  const { container: renderContainer } = render(
      <Router>
        <SignUp />
      </Router>
    );
});
container = renderContainer;
```